tests/test_plugins.py

```diff
@@ -248,7 +248,7 @@ def test_load_extra_entrypoints_missing_attribute(capsys):
     )
 
 
-def test_plugins_load_on_prefect_package_init(monkeypatch):
+def test_plugin_load_on_prefect_package_init(monkeypatch):
```

Comment:
```suggestion
def test_plugin_loads_on_prefect_package_init(monkeypatch):
```

Reply:
I changed this intentionally. We're no longer asserting the plugins load, we're testing plugin loading in general.